### PR TITLE
Fix riot router

### PR DESCRIPTION
**Pull request by taka-sho**

### CODE REVIEW

File: test/riot-route.test.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as assert from 'power-assert'
+
+describe.only('riot-route', function () {
```
Comment (9renpoto) [inline test/riot-route.test.ts]: remove only flag

**Comment by 9renpoto** [comment]
Thank you, please check location.href !

File: test/riot-route.test.ts (hunk 1 above)
Comment (taka-sho) [inline test/riot-route.test.ts]: oops

**Comment by 9renpoto** [comment]
[](http://tiqav.com/1Ps)

**Comment by 9renpoto** [comment]
[孤独のグルメ](http://tiqav.com/vQ)

**Pull request merged**